### PR TITLE
Upgrade nunit3 from 3.6.1 to 3.7.1

### PR DESCRIPTION
**Pull request by LiyueWang**

### CODE REVIEW

File: TestAssembly.AssemblyLevelCategorization/packages.config [hunk 1]
```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (JeffAshton) [inline TestAssembly.AssemblyLevelCategorization/packages.config]: Why was this deleted?

File: TestAssembly.NUnit3/packages.config [hunk 2]
```diff
@@ -1,4 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (JeffAshton) [inline TestAssembly.NUnit3/packages.config]: Why was this deleted?

Comment (LiyueWang) [inline TestAssembly.NUnit3/packages.config]: Because I find there are too many places to upgrade the version NUnit... and we have the nunit package om package.config in `NUnitTestCategoryAssertor`, so it should be OK to remove them?

Comment (JeffAshton) [inline TestAssembly.NUnit3/packages.config]: If you did it manually, then not ideal.  Visual studio uses these to manage updates in the future.  If you did it manually, i would say revert

Comment (JeffAshton) [inline TestAssembly.NUnit3/packages.config]: And use the built in visual studio functionality
